src/backend/joanie/core/views.py

```diff
@@ -15,7 +15,7 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["title"] = "👨‍💻Development email preview"
         context["email"] = order.owner.email
-        context["username"] = order.owner.username
+        context["username"] = order.owner.get_full_name() or order.owner.username
```

Comment:
is it better to also modify the context value name from "username" to "full_name" (in the template too)
if maybe one day we will need to display the "name" and the "username" in the same email